### PR TITLE
All-in-one Benchmark run.py: Ignore error if import BenchmarkWrapper …

### PR DESCRIPTION
**Pull request by ATMxsp01**
Ignore error if import BenchmarkWrapper failed.

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -32,7 +32,10 @@
 import sys
 sys.stdout.reconfigure(encoding='utf-8')
 
-from ipex_llm.utils import BenchmarkWrapper
+try:
+    from ipex_llm.utils import BenchmarkWrapper
+except:
+    pass
```
Comment (Oscilloscope98) [inline python/llm/dev/benchmark/all-in-one/run.py:38]: Should we fix the `except` logic for npu-related tesg api?

**Comment by plusbang** [comment]
Currently, no benchmarkwrapper is compatible for transformers==4.47.0. Need to support it later : )

**Review by Oscilloscope98 — APPROVED** [review]
Others LGTM

File: python/llm/dev/benchmark/all-in-one/run.py (hunk 1 above)
Comment (plusbang) [reply python/llm/dev/benchmark/all-in-one/run.py:38]: > Should we only use the `except` logic for npu-related test_api?

But it's not quite convenient to add test API related check here. This is only a work around, we could remove it after support.

**Pull request merged**